mlrun/projects/pipelines.py

```diff
@@ -585,12 +628,12 @@ def run(
         return _PipelineRunStatus(run_id, cls, project=project, workflow=workflow_spec)
 
     @staticmethod
-    def wait_for_completion(run_id, project=None, timeout=None, expected_statuses=None):
-        if timeout is None:
-            timeout = 60 * 60
+    def wait_for_completion(run, project=None, timeout=None, expected_statuses=None):
+        logger.info("Waiting for pipeline run completion")
```

Comment:
Looks more like a debug log and it would be also beneficial to add some information about run. For instance, project and run_id.

Reply:
About debug log - agree with you but I just copied the same behavior that was previously.
This log is show to the user so I think I should remain the log level but as you suggest I will add the project and run_id.